canvas_modules/common-canvas/src/common-properties/actions/image/image.jsx

```diff
@@ -56,12 +56,17 @@ class ImageAction extends React.Component {
 
 		const image = (
 			<div data-id={this.props.action.name}>
-				<img
-					src={this.props.action.image.url}
+				<button
+					className="properties-action-image-button"
```

Comment:
@matthoward366 I am still working on this PR. Is it okay if we include button like this or should this be a Carbon button ?

Reply:
Seems fine as long as the UI looks the same.

Doesn't carbon have attributes for rendering icon only buttons?

Reply:
Yes, carbon has rendering icon only buttons.
If we include `hasIconOnly` it shows up carbon tooltip on focus. Is it okay to replace custom tooltip with carbon tooltip ?


https://github.com/user-attachments/assets/8606cb3a-7682-4318-80ad-bd0e2470d419

Reply:
Currently the custom tooltip isn't triggered on buttons. Is this fine or should this be created as a new issue ?

Action Buttons


https://github.com/user-attachments/assets/3bf3a989-b321-4652-8977-69eb499e099e